tests/scripts/tesseract.sh

```diff
@@ -1,15 +1,19 @@
 #!/bin/bash
 
-mkdir ~/tesseract
-cd ~/tesseract
-wget http://alpha.library.yorku.ca/tesseract.tar.gz
-wget http://alpha.library.yorku.ca/tessdata.tar.gz
-tar xf tesseract.tar.gz && rm -rf tesseract.tar.gz
-dpkg -i tesseract/libleptonica.deb
-dpkg -i tesseract/tesseract-ocr.deb
-tar xf tessdata.tar.gz && rm -rf tessdata.tar.gz
-cp tessdata/* /usr/local/share/tessdata/
-sudo ldconfig
-echo -e "\ntesseract output:"
+if [[ $TRAVIS_PHP_VERSION == '5.3.3' ]]; then
+  mkdir ~/tesseract
+  cd ~/tesseract
+  wget http://alpha.library.yorku.ca/tesseract.tar.gz
+  wget http://alpha.library.yorku.ca/tessdata.tar.gz
+  tar xf tesseract.tar.gz && rm -rf tesseract.tar.gz
+  dpkg -i tesseract/libleptonica.deb
+  dpkg -i tesseract/tesseract-ocr.deb
+  tar xf tessdata.tar.gz && rm -rf tessdata.tar.gz
+  cp tessdata/* /usr/local/share/tessdata/
+  sudo ldconfig
+  echo -e "\ntesseract output:"
```

Comment:
I think this was meant as context for the `tesseract --version && tesseract --list-langs && cd ~ && rm -rf ~/tesseract` line, so perhaps moving it after the `if ... else .. fi` makes sense.

Reply:
Right! You are the best 👍 Doing it now